typescript/packages/subsurface-viewer/src/layers/grid3d/grid3dLayer.ts

```diff
@@ -85,8 +85,17 @@ async function loadData<T extends TTypedArray>(
         return new type(data);
     }
     if (typeof data === "string") {
-        const stringData = await load(data as string, JSONLoader);
-        return new type(stringData);
+        const extension = data.split(".").pop()?.toLowerCase();
```

Comment:
Add comments for expected data (json file, or ??)

Reply:
Comment added